R/util.R

```diff
@@ -112,6 +112,15 @@ is_null_or_string <- function(text) {
   is.null(text) || (is.character(text) && (length(text) == 1))
 }
 
+# TODO: Export knitr:::is_blank() in xfun, and use here
```

Comment:
Done: https://github.com/yihui/xfun/commit/6f4eddd0d74974a12bbc431053849495ac6bd1bc

Reply:
Thanks ! 

Should we use it now here ? Or usually, you go through this kind of TODO after a new **xfun** is released ?

Reply:
No, not now, but after the next **xfun** release.

Yes, I search for "TODO" in each package before I start thinking about making a CRAN release of that package.

Reply:
Perfect. I'll try to remember that ! 👍

Reply:
Usually a TODO comment is not urgent to me, and can be done at any time in the future (not necessarily before the next release) if I remember.